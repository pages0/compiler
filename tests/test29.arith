let lst: [int] = (1 :: 2 :: 3 :: 4 :: [] : int)
in  (hd lst, tl lst)