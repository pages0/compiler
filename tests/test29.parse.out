( let [ lst ( Cons 1 ( Cons 2 ( Cons 3 ( Cons 4  [] : int ) ) ) ) ] ( pair ( hd lst ) ( tl lst ) ) )